lib/pbench/server/cache_manager.py

```diff
@@ -909,16 +909,16 @@ def get_unpacked_size(self) -> int:
         if not self.dataset:
             return 0
 
-        size = Metadata.getvalue(self.dataset, Metadata.SERVER_UNPACKED)
-        self.logger.info("SIZE unpacked = {}", size)
+        source = Metadata.SERVER_UNPACKED
+        size = Metadata.getvalue(self.dataset, source)
         if not size:
             try:
-                size = int(
-                    Metadata.getvalue(self.dataset, "dataset.metalog.run.raw_size")
-                )
-                self.logger.info("SIZE metalog = {}", size)
+                source = "dataset.metalog.run.raw_size"
+                size = int(Metadata.getvalue(self.dataset, source))
             except (ValueError, TypeError):
+                source = None
                 size = 0
+        self.logger.info("{} unpacked size (from {}) = {}", self.name, source, size)
```

Comment:
Should this be `info()` or `debug()`?  Ditto line 1012.

Reply:
Right now we have two `info` messages for each unpack with the value from `server.unpacked-size` (if any) and `dataset.metalog.run.raw_size`, but we never report the actual unpacked size after. Having looked at a lot of these in the log, I got tired of the two messages up front (and reworked them into one), and missed having the actual size after.

Whether we might eventually want to remove or downgrade these is a separate question that could be applied to a lot of our observability stack ... some day.